mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayPlainDTOToWayPreview.ts

```diff
@@ -9,7 +9,7 @@ import {WayPreview} from "src/model/businessModelPreview/WayPreview";
  */
 export const wayPlainDTOToWayPreview = (wayDTO: SchemasWayPlainResponse): WayPreview => {
   const status = getWayStatus({
-    status: wayDTO.isCompleted ? "Completed" : null,
+    status: wayDTO.isCompleted ? "completed" : null,
```

Comment:
let's don't use just string 

we have enum WayStatus.completed

Reply:
done